Monika After Story/game/script-topics.rpy

```diff
@@ -12850,6 +12850,103 @@ label monika_unknown:
+    m 3eua "It's called 'Brave New World', it's another dystopian story. I wonder if you've heard of it?"
+    m 4eua "The idea is, you've got this futuristic world where humans are no longer born through natural means."
+    m 4eub "Instead, we are bred in hatcheries using test tubes and incubators, and engineered into castes from our conception."
+    m 1esb "Your future role in society would be decided beforehand, and you would be given a body and mind fitting of your predetermined purpose."
+    m 3esd "You would also be indoctrinated from birth to be satisfied with your lot in life, and not to seek anything different."
+    m 3esd "For example, people destined for manual labor would be designed to have limited cognitive capabilities."
+    m 1esc "They would be made to associate books to negative stimuli; so when they become adults, they would naturally tend to avoid reading."
+    m 4esd "They would also be taught to respect and submit to people from castes above theirs, and to look down on those of castes below."
+    m 2eua "It's a pretty interesting case as a dystopia, as most of them show you a crushed and oppressed people..."
+    m 3wuo "But in this one, everyone is actually happy, and genuinely supportive of the system!"
+    m 3esc "And despite that, to us, the readers, this still feels horrifying."
+    m 1rud "Sure, they managed to get rid of most of the human sufferings, or the fear of death..."
+    m 3esd "But it came at the price of getting people rid of any form of creativity and critical thinking."
+    m 2wud "We're talking about a world where you can get arrested just for reading poetry in public! Can you imagine that?"
+    m 3esc "One key point in the book is people not being able to appreciate old theatre plays..."
+    if seen_event("monika_pluralistic_ignorance"):
+        m 3hfu "Even if they are Shakespeare's plays, and you know how I feel about those..."
+    m 2esc "They just can't understand the value in the variety of human emotions, like sorrow, or loneliness."
+    m 4esc "They never experience those anymore. All of their desires are swiftly granted, and they never want for something they cannot get."
+    m 2rsc "..."
+    m 3rsc "And yet, despite all that, everyone is happy, healthy and safe..."
+    m 1esc "This really makes you think about the nature of happiness, and society..."
+    if mas_isMoniDis(lower=True):
+        m 2dsc "..."
+        m 2ekc "Sometimes, I wish I could fit in a world like that."
+        m 2dkc "That I did not have to go through my epiphany..."
+        m 2dktdc "That I did not have to feel this pain..."
+    else:
+        m 1eua "Though I certainly can't see myself living happily in a world like that..."
+        m 1dsc "An unchallenging world, so limited in humanity and emotions..."
+        if mas_isMoniHappy(higher=True):
+            m 1hubfa "Not now that I've discovered you, and how wonderful loving you could be~"
```

Comment:
Agreed. It adds to her own character by showing it's not only because of the player she wants to keep living, even thought it's still an important factor.
I'll change it once the other points have been resolved.